Projects/l0/Controller_motor/lib/Controller_motor/controller_motor.c

```diff
@@ -629,8 +626,7 @@ void HAL_SYSTICK_Motor_Callback(void)
         last_systick = HAL_GetTick();
     }
     // ****** Linear interpolation *********
-    if ((motor.mode.mode_angular_position || motor.mode.mode_linear_position) &&
-        (motor.mode.mode_angular_speed || motor.mode.mode_linear_speed))
+    if ((motor.mode.mode_angular_position || motor.mode.mode_linear_position) && (motor.mode.mode_angular_speed || motor.mode.mode_linear_speed))
```

Comment:
Perhaps we should put OR and AND operators at line beginning.  So, after clang format processing, the conditions should be on different lines.